include/global.h

```diff
@@ -173,6 +173,13 @@ typedef void (*VoidFn)(void);
     a = ((u8)b ^ (u8)a) * -1;                                                           \
     b = (u8)b * -1;
 
+#define NEGATE(var)                                                                     \
+    ({                                                                                  \
+        s32 temp = var;                                                                 \
+        var = -temp;                                                                    \
+    })
+#define DIRECT_NEGATE(var) (var = -var;)
```

Comment:
Do we really need this? I can understand `NEGATE` if we are using it for non matches?

Reply:
Kinda, sorta, I guess?
I just noticed that everywhere this way was specifically used and (var *= -1) didn't usually work.
Could rename to NEGATE or remove entirely. One of the things I just left in and waited on your input! :D